code/game/objects/items/devices/radio/electropack.dm

```diff
@@ -74,6 +74,8 @@
 		var/mob/M = loc
 		var/turf/T = M.loc
 		if(istype(T, /turf))
+			if(M.stat == DEAD)
+				return
```

Comment:
this means if you're shocking dead people they won't spark either which is kinda harmless, change the check below